### ISSUE TITLE
Change the release workflow

### ISSUE THREAD

**Issue by sanacl** [chaoss/grimoirelab-sirmordred#403]
Hey @valeriocos , I've been talking to @sduenas after the issues with the missing changes in the `Dockerfile` and `stage` files for the past releases. To give some context to the audience: I created a PR wich changes in the /docker files which were merged to master, but when the release was created, those changes were not used and we've been in that error-loop for several iterations/releases. 

My understanding is when you create a release, you copy the branch of the previous release. If this is correct, could be do it from master? this way we just care about having master branch updated and the new releases can create a branch based on master if needed. 

So, to sum up: can we change the release workflow and start using `master` branch as reference for all files (Docker files included)?

**Comment by valeriocos** [comment]
Yes, it makes sense. I don't know why the current procedure is to use the branch of the previous release

**Comment by valeriocos** [comment]
This issue is related to https://github.com/chaoss/grimoirelab-sirmordred/issues/376

**Comment by sanacl** [comment]
> This issue is related to #376

good catch

**Comment by valeriocos** [comment]
The release workflow has changed. Now we use the master branch
